### DISCUSSION TITLE
Paperless is not "consuming" files

### DISCUSSION THREAD

**Discussion by flexxipanda** [jonaswinkler/paperless-ng#1219] [Support]
Hi guys,

I've setup an installed paperless-ng but it does not "consume" the files I put in the consume folder. Basically nothing happens.

I've set it up in a Ubuntu VM using the docker install script.

I mounted a SMB network share to /home/kev/NetShare2/ and in there are two folders "Scan" and "Archiv" which are set to the consume and media folders. 

Paperless-ng is installed and running under the user "kev".

I don't know if it is maybe permission problem but it looks alright to me. 

> kev@kev-VirtualBox:~$ ls -l
>drwxr-xr-x 2 kev kev    0 Aug  6 23:12 NetShare2

I also tested to manually delete and save files in there, and it works.

The dockers are also running: https://imgur.com/a/9TdAlfw

The webserver is also running and I can acces it from other devices on the network.

My docker-compose.yml file looks like this: https://pastebin.com/V8KNvVbF and the docker-compose.env like this: https://pastebin.com/pR8TF4fJ (Is it supposed to be that empty?)

Did I do something wrong somewere? Do I miss something? I hope you can help me.

**Comment by davidmlafuente** [comment]
I have the same issue here.

**Reply by flexxipanda** [reply]
I found a solution.  I found the install script to not really work at all. So, I did a clean install of ubuntu and tried it with [this guy's install instructions](https://blog.prutser.net/2020/12/15/manage-your-mail-with-machine-learning-and-paperless-ng/). He's talking about installing it directly to docker but it works with ubuntu as well. With that I managed to make it work with the default consume folder, but after I changed that in the .yml file it stopped working again. Apparently, if you make changes you to need to update it with `docker-compose up -d`. (Maybe I also ran `docker-compose pull` again before, I can't remember anymore.)
